ui/packages/components/src/Link/Link.tsx

```diff
@@ -13,6 +14,7 @@ type CustomLinkProps = {
   iconBefore?: React.ReactNode;
   iconAfter?: React.ReactNode;
   arrowOnHover?: boolean;
+  target?: HTMLAttributeAnchorTarget | undefined;
```

Comment:
Target is not exposed by the next link props we expose...and I don't want to spend an hour unpacking the bizarro type signature they are exporting to properly bubble up native anchor props...so I just added it.